kubernetes/client/models/admissionregistration_v1_service_reference.py

```diff
@@ -13,7 +13,6 @@
 import pprint
 import re  # noqa: F401
 
-import six
```

Comment:
The file is generated by openapi-generator. The change needs to be made upstream. Otherwise the change will be override when we re-generate the client

Reply:
Thanks. I think it's already gone. https://github.com/OpenAPITools/openapi-generator/pull/16675
Wait & see